exercises/bob/example.go

```diff
@@ -2,29 +2,32 @@ package bob
 
 import "strings"
 
-const testVersion = 3
-
-func Hey(drivel string) string {
-	switch drivel = strings.TrimSpace(drivel); {
-	case silent(drivel):
+// Hey returns Bob's responses to any given dialogue
+func Hey(dialogue string) string {
```

Comment:
I like dialogue. I think in my Ruby one I ended up with `remark`, since dialogue implies two sides, but that's some level five nitpickery so I don't think we need to change it here.